src/integrations/Braze/browser.js

```diff
@@ -150,94 +93,102 @@ class Braze {
    * or the updated fields.
    * @param {*} rudderElement
    */
+
+  // eslint-disable-next-line sonarjs/cognitive-complexity
   identify(rudderElement) {
-    logger.debug("in Braze identify");
-    const { userId } = rudderElement.message;
-    const { address } = rudderElement.message.context.traits;
-    const birthday =
-      rudderElement.message.context.traits?.birthday ||
-      rudderElement.message.context.traits?.dob;
-    const { email } = rudderElement.message.context.traits;
-    const firstname =
-      rudderElement.message.context.traits?.firstname ||
-      rudderElement.message.context.traits?.firstName;
-    const { gender } = rudderElement.message.context.traits;
-    const lastname =
-      rudderElement.message.context.traits?.lastname ||
-      rudderElement.message.context.traits?.lastName;
-    const { phone } = rudderElement.message.context.traits;
+    logger.debug('in Braze identify');
+    const { message } = rudderElement;
+    const { userId } = message;
+    const {
+      context: {
+        traits: {
+          email,
+          firstName,
+          firstname,
+          lastname,
+          lastName,
+          gender,
+          phone,
+          address,
+          birthday,
+          dob,
+        },
+      },
+    } = message;
```

Comment:
Doesn't this crash if message doesn't have `traits` property?

Reply:
Yeah correct need to do conditional destructuring let me fix it

Reply:
Addressed